prow/plugins/label/label_test.go

```diff
@@ -387,6 +394,7 @@ func TestLabel(t *testing.T) {
 			expectedNewLabels:     []string{},
 			expectedRemovedLabels: []string{},
 			commenter:             orgMember,
+			expectedBotComment:    true,
 		},
 		{
```

Comment:
Please add a test case for `/label foo` where `foo` is not a label in the repo and a test case where `foo` is a label in the repo, but is not configured as an additional label that can be applied with the `/label` command.

Reply:
Do lines #400 and #410 address this ask?

Reply:
I see your point. Thank you! Making changes.